engine/src/w32flst.cpp

```diff
@@ -73,6 +73,7 @@ uint2 weighttable[] =
         FW_DONTCARE,
         FW_THIN,
         FW_EXTRALIGHT,
+		FW_EXTRALIGHT,
```

Comment:
It does seem to be - there are 10 MCFW_ constants so there should be 10 entries in this array too.